.pylintrc

```diff
@@ -58,7 +58,7 @@ ignore-paths=^.*\\generated_models\\.*$|^.*/generated_models/.*$
 # Files or directories matching the regex patterns are skipped. The regex
 # matches against base names, not paths. The default value ignores Emacs file
 # locks
-ignore-patterns=venv,.venv
+ignore-patterns=venv,.venv,jupyter_server_config.py
```

Comment:
Question: is pylint a part of pre-commit? pre-commit has been green :thinking:

Reply:
Yes, the pre-commit hook always failed for me due to this file, but it only runs of `.py` files are modified so you might not have observed the failure in a while? not sure.

Reply:
In CI we run `pre-commit run --all-files` which I thought means running all `pre-commit` against all files.

Reply:
I think this depends on the specific pre-commit hook, but yes I would argue gitignored files should usually not be checked by pre-commit. In this case even after digging I am not sure what is going on, but this anyway fixes it and doesnt hurt ;)